components/brave_wallet_ui/components/desktop/views/portfolio/components/accounts-and-transctions-list/index.tsx

```diff
@@ -74,15 +76,15 @@ const AccountsAndTransactionsList = (props: Props) => {
     onSpeedupTransaction
   } = props
 
-  const getBalance = useBalance(selectedNetwork)
+  const getBalance = useBalance(networkList)
 
   const findAccount = (address: string): WalletAccountType | undefined => {
     return accounts.find((account) => address === account.address)
   }
 
   const accountsList = React.useMemo(() => {
     if (selectedAsset?.isErc721) {
-      return accounts.filter((account) => Number(account.balance) !== 0)
+      return accounts.filter((account) => Number(account.nativeBalanceRegistry[selectedNetwork.chainId]) !== 0)
```

Comment:
Possible `NaN` value here. Maybe we should handle it?

Reply:
Added a check!